models/migrations/v213.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"xorm.io/xorm"
+)
+
+func createRepoSettingsTable(x *xorm.Engine) error {
+	type RepoSetting struct {
+		ID           int64  `xorm:"pk autoincr"`
+		RepoID       int64  `xorm:"index unique(key_repoid)"`              // to load all of someone's settings
+		SettingKey   string `xorm:"varchar(255) index unique(key_repoid)"` // ensure key is always lowercase
+		SettingValue string `xorm:"text"`
+	}
+	if err := x.Sync2(new(RepoSetting)); err != nil {
+		return fmt.Errorf("sync2: %v", err)
+	}
+	return nil
```

Comment:
I think we can already guess which method failed if a migration as simple as that fails...
```suggestion
	return x.Sync2(new(RepoSetting))
```